lib/couchbaseX.js

```diff
@@ -352,7 +352,17 @@ class CouchbaseAccessor extends Accessor {
         // err: query error count, e.g "Error: 2"
         // res: { id1: { value }, id2: { value }, id3: { error: { CouchbaseError: 'The key does not exist...' } } }
         return resolve(Object.keys(res)
-          .map(id => res[id].value ? [id, this.unpackRes(res[id].value)] : null)
+          .map(id => {
+            const item = res[id];
+            if (!item || !item.cas) {
+              return null;
+            }
+
+            const value = item.value || {};
+            value._cas = item.cas;
+
+            return [id, value];
```

Comment:
can you simplify the code a bit more, better to have a test code to cover the data transformation.